app_dart/lib/src/request_handlers/file_flaky_issue_and_pr.dart

```diff
@@ -36,18 +38,24 @@ class FileFlakyIssueAndPR extends ApiRequestHandler<Body> {
     final BigqueryService bigquery = await config.createBigQueryService();
     final List<BuilderStatistic> builderStatisticList = await bigquery.listBuilderStatistic(kBigQueryProjectId);
     final YamlMap? ci = loadYaml(await gitHub.getFileContent(slug, kCiYamlPath)) as YamlMap?;
+    final pb.SchedulerConfig unCheckedSchedulerConfig = pb.SchedulerConfig()..mergeFromProto3Json(ci);
+    final CiYaml ciYaml = CiYaml(
```

Comment:
```suggestion
    final Commit totCommit = await generateTotCommit(slug: commit.slug, branch: Config.defaultBranch(commit.slug));
    final CiYaml ciYaml = await scheduler.getCiYaml(totCommit);
``

You'll need to update the constructor to pass `Scheduler`.

Reply:
Discussed with @CaseyHillers , this is going to be moved to another PR, since there are complications with the test setup.